opentech/static_src/src/app/src/containers/GroupByRoundDetailView.js

```diff
@@ -45,13 +46,11 @@ class GroupByRoundDetailView extends React.Component {
 }
 
 const mapStateToProps = (state, ownProps) => ({
-    isErrored: getRoundsErrored(state),
-    errorMessage: getByGivenStatusesError(ownProps.submissionStatuses)(state),
```

Comment:
@chris-lawton This was returning a boolean, so moved to the `isErrored` prop.

Reply:
https://github.com/OpenTechFund/opentech.fund/commit/e7ef88d7cddbaf63ccf926801d9200dabb705b2a#diff-97f00faee6202c788f86bcc7b8eefe56R49

You will need to rebase. I've added a quick hack.